crates/pop-parachains/src/templates.rs

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: GPL-3.0
+#[derive(Debug, Clone, PartialEq)]
+pub struct Config {
+	pub symbol: String,
+	pub decimals: u8,
+	pub initial_endowment: String,
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub enum Template {
+	// Pop
+	Base,
```

Comment:
`BasePop` maybe consistent with the naming scheme of the other two variants `Partiy*`  ?

Reply:
No, Pop are the default ones of the tool: pop base, pop contracts, pop evm and pop nfts. 
ParityContracts is called like this only because Contracts will exist with Pop as a provider